### PR TITLE
Added a tutorial on Claiming Rooms

### PR DESCRIPTION
**Pull request by MrabEzreb**
See title

### CODE REVIEW

File: source/contributed/claiming_rooms.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+The first step is to claim the room! Use a spawn to create a creep. The creep needs a body made of 1 MOVE part and 1 CLAIM part. Next, move the creep into the room you need to claim. Finally, claim the room. Let's break this down.
+### Step 1.1: Create a Claimer Creep
+I will assume that you have completed step 0, which would mean you know how to create a creep. Make a creep with a body ```[MOVE, CLAIM]```, put the ID of the target room into it's memory under ```claimTarget```, and set its name to ```claimer1```
+
+### Step 1.2: Get to the room you want to claim
+If you haven't claimed a room, it will not be in your ```Game.rooms``` object. That means you can't tell your creep to move to ```Game.rooms["ROOM_ID"].controller```, because ```ROOM_ID``` isn't in ```Game.rooms```. So how do you get there?  
+We are going to get the room that our creep is in right now, then find an exit in the direction of the target room, then move to the closest exit.
+
+    //Get the claimer1 creep
+    var creep = Game.creeps.claimer1
```
Comment (Spedwards) [inline source/contributed/claiming_rooms.md]: Please never encourage the use of `var`. Always use `let` or `const`.

Comment (MrabEzreb) [inline source/contributed/claiming_rooms.md]: ok, fixing

Comment (Spedwards) [inline source/contributed/claiming_rooms.md]: A couple were missed in your for-loops, but thank you.

Comment (MrabEzreb) [inline source/contributed/claiming_rooms.md]: ok, switching those as well

**Comment by MrabEzreb** [comment]
Just checking, but the contributed articles section of the wiki mentioned 14 days of community review. Is there any reason why I haven't gotten a PR on my fork for a month and a half?

**Comment by artch** [comment]
Thank you for your efforts, but this article doesn't seem to be useful enough to participate in the Contributed Articles program. Such basic things as claiming rooms and building them is considered to be discovered by players on their own. Also, the article contains sub-optimal solutions like using `findExitTo` instead of `findPathTo(new RoomPosition(...))`.

**Pull request closed without merging**